src/main/java/bit/javaoop/Company.java

```diff
@@ -0,0 +1,45 @@
+package bit.javaoop;
+
+public class Company {
+
+    private String name;
+    private Ceo ceo;
+
+    public Company(String name, Ceo ceo) {
+        this.name = name;
+        this.ceo = ceo;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Ceo getCeo() {
+        return ceo;
+    }
+
+    public void hireCeo(Ceo ceo) {
+        this.ceo = ceo;
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append(ceo.toString());
+        stringBuilder.append("\n");
+
+        for(AbstractManager manager : ceo.getManagers()) {
+            stringBuilder.append("\t");
+            stringBuilder.append(manager.toString());
+            stringBuilder.append("\n");
```

Comment:
- Builder to taki fajny paterrn w którym jak implementujemy metody (zazwyczaj ustawiające jakieś pola) na końcu robimy return this; (zwracamy samego siebie) co pozwala nam na coś takiego:
```
stringBuilder
                    .append("\t")
                    .append(manager.toString())
                    .append("\n");
```
- \+ za użycie [escape sequences](https://docs.oracle.com/javase/tutorial/java/data/characters.html)